### ISSUE TITLE
[Bug] GetObject or any reflect Get operation failing for a struct inside a struct for all omitempty fields. 

### ISSUE THREAD

**Issue by kartavya-ramnani** [aerospike/aerospike-client-go#301]
If there a struct inside a struct which has fields with omitempty tag, that field NEVER gets fetched. 

For example :

struct A {
 Bstruct B `as:"b"`
}

struct B {
Test string `as:"test,omitempty"` // this will never be returned in any read reflection commands
}

Reason :

Inside read_command_reflect.go :
setStructValue :
alias will be Test
Then this line comes :
```
tag := strings.Trim(fld.Tag.Get(aerospikeTag), " ")
```
and tag is test,omitempty
so alias becomes test,omitempty 

And valMap[alias] will fail. 

This is a major bug in my opinion and I can really use help in getting this fixed asap.

**Comment by kartavya-ramnani** [comment]
Fix :
Replace in read_command_reflect.go -> setStructValue :
```
tag := strings.Trim(fld.Tag.Get(aerospikeTag), " ")
```
with
```
tag := strings.Trim(stripOptions(fld.Tag.Get(aerospikeTag)), " ")
```

**Comment by kartavya-ramnani** [comment]
[Commit](https://github.com/aerospike/aerospike-client-go/commit/f569bd48c9b02fa5424987e8d78bf4a27973ca1e) which may have introduced this.

**Comment by khaf** [comment]
I tried to fix it today, but this issue turned out to be related to a bigger problem that I have to address. Will release the fix next week.

**Comment by kartavya-ramnani** [comment]
Can I ask what is the bigger problem ? @khaf 
We are using this library on high production traffic.

**Comment by khaf** [comment]
The PR that introduced this bug introduced another regression that tests didn't catch, and now I have to find a way to fix it in a way that is backwards compatible without breaking the internet.

**Comment by kartavya-ramnani** [comment]
Hi @khaf  : Is this fixed ?

**Comment by khaf** [comment]
Not yet; I'm now considering releasing a quick fix for you and spend a bit more time for the other one.

**Comment by kartavya-ramnani** [comment]
@khaf  : Will appreciate it if you do that.

**Comment by khaf** [comment]
Fix released in v3.0.1. Thank you for your patience.
